.circleci/config.yml

```diff
@@ -15,9 +15,48 @@ jobs:
           key: yarn-packages-{{ checksum "yarn.lock" }}
           paths:
             - ~/.cache/yarn
+      - persist_to_workspace:
+          root: /home/circleci
+          paths:
+            - project
+
+  test:
+    docker:
+      - image: circleci/node:10
+    steps:
+      - attach_workspace:
+          at: /home/circleci
       - run: yarn lint --no-fix
       - run: yarn lint:pug
       - run: yarn lint:style
       - run: yarn test:unit --coverage --coverageReporters html
       - store_artifacts:
           path: coverage/
+
+  deploy:
+    docker:
+      - image: circleci/node:10
+    steps:
+      - attach_workspace:
+          at: /home/circleci
+      - run: yarn build:demo
+      - add_ssh_keys:
+          fingerprints:
+            - "eb:dc:cd:0a:f4:99:56:3f:79:f7:22:a2:6c:21:56:c7"
+      - deploy:
+          command: .circleci/pages-deploy.sh
```

Comment:
This job only runs on master branch.  However, it might be good to double-check this with bash's `if [ $env == 'master' ] ...`

I don't think that's necessary for something like this though.